spec/components/deferral_details/view_spec.rb

```diff
@@ -41,7 +39,7 @@ module DeferralDetails
       let(:trainee_stub) { Trainee.new }
 
       it "renders the deferred before course start date message" do
-        expect(component).to have_text(strip_tags(t("deferral_details.view.deferred_before_starting")))
+        expect(component).to have_text("Trainee deferred before their ITT started")
```

Comment:
Could we continue to use `strip_tags()`? Saves us from having duplicate text.

Reply:
Fixed that, was an artifact from my rebase